app/src/lib/stats/stats-store.ts

```diff
@@ -1777,6 +1798,54 @@ export class StatsStore implements IStatsStore {
     }))
   }
 
+  // Generates the stat field name for the given PR review type and suffix.
+  private getStatFieldForRequestReviewState(
+    reviewType: IAPIPullRequestReview['state'],
+    suffix: PullRequestReviewStatFieldSuffix
+  ): PullRequestReviewStatField {
+    let infix: PullRequestReviewStatFieldInfix
+
+    if (reviewType === 'APPROVED') {
+      infix = 'Approved'
+    } else if (reviewType === 'CHANGES_REQUESTED') {
+      infix = 'ChangesRequested'
+    } else {
+      infix = 'Commented'
+    }
+
+    return `pullRequestReview${infix}${suffix}`
+  }
```

Comment:
Not very fond of `IAPIPullRequestReview['state']`, not only because it means bringing API-related stuff here (and some other interfaces), but also because that field has more values than we need for notifications.

Maybe I should write something like a `ValidNotificationPullRequestReview` type which is like `IAPIPullRequestReview` but where state is either `APPROVED`, `CHANGES_REQUESTED` or `COMMENTED`, and a type guard to ensure this 🤔  I'll give that approach a try.

Reply:
Done in 6156fe9cd2ce16472ded4194ca76d7c727f692eb...2ae88489687a033235759d17aa5d2784a95f1945

Reply:
I moved that cleanup to #14175